tests/simulate/test_lammps.py

```diff
@@ -578,6 +616,39 @@ def test_writetrajectory(self):
             self.assertCountEqual(snap.typeid, [1, 1, 2, 2, 2])
             self.assertCountEqual(snap.mass, [1, 1, 1, 1, 1])
 
+    def test_writetrajectory_gsd(self):
+        """Test write trajectory simulation operation."""
+        ens, pot = self.ens_pot()
+        init = relentless.simulate.InitializeRandomly(
+            seed=1, N=ens.N, V=ens.V, T=ens.T, diameters={"1": 1, "2": 1}
+        )
+        analyzer = relentless.simulate.WriteTrajectory(
+            filename="test_writetrajectory.gsd",
+            every=100,
+            velocities=True,
+            images=True,
+            types=True,
+            masses=True,
+        )
+        lgv = relentless.simulate.RunLangevinDynamics(
+            steps=500,
+            timestep=0.001,
+            T=ens.T,
+            friction=1.0,
+            seed=1,
+            analyzers=[analyzer],
+        )
+        h = relentless.simulate.LAMMPS(init, operations=lgv, executable=self.executable)
+        sim = h.run(potentials=pot, directory=self.directory)
+
+        # read hoomd trajectory file
+        file = sim.directory.file("test_writetrajectory.gsd")
+        with gsd.hoomd.open(file) as traj:
+            for snap in traj:
+                self.assertEqual(snap.particles.N, 5)
+                self.assertIsNotNone(snap.particles.velocity)
```

Comment:
Is there a reason you dropped some of the other things we tested in the other test? Did it not work right?

Reply:
I had it test the same things that are tested when writing a gsd from HOOMD.

Reply:
Oh OK, I wonder why we didn't test for that in the HOOMD GSD?